spec/acceptance/api/account_spec.rb

```diff
@@ -14,7 +14,7 @@
     account
   end
 
-  let(:expected_provider_fields) { %w[admin_domain domain admin_base_url base_url from_email support_email finance_support_email site_access_code] }
+  let(:expected_provider_fields) { %w[domain admin_base_url base_url from_email support_email finance_support_email site_access_code] }
```

Comment:
why remove this?

Reply:
Because the test fails now that `admin_domain` is private.

Reply:
Here is my understanding, correct me if I'm wrong. This is testing our public API. After changing method to `private` it is no longer present. If this is the case, I think we need to update our representer to include this field but by calling the non-deprecated method `#internal_admin_domain`. Or not?

Reply:
Yes, makes sense